README.md

```diff
@@ -15,6 +15,15 @@ By default, scrubs:
 - Social Security numbers (SSNs)
 - passwords in URLs
 
+To disable any of the default scrubs above, use:
```

Comment:
Actually not sure should we fold in the IP option here?

Reply:
IP scrubbing is false by default; I wanted to preserve the conventions that already exist so, if our PR into the master repo is accepted and users upgrade to the new version, nothing breaks. If this was a new gem (or if we don't want our change merged into master) - I agree that it's better to treat all flags uniformly